src/app/routes/connector-ui/asset-page/asset-edit-dialog/asset-edit-dialog.component.html

```diff
@@ -82,17 +103,26 @@ <h1 mat-dialog-title>
           <!-- Language -->
           <language-select
             class="grow"
-            label="Language"
+            label="Language of the content"
             [control]="form.metadata.controls.language"></language-select>
 
           <div class="flex flex-row space-x-[10px]">
             <!-- Content Type -->
-            <mat-form-field class="grow">
+            <mat-form-field class="grow mb-2">
               <mat-label>Content Type</mat-label>
               <input
                 matInput
                 [formControl]="form.metadata.controls.contentType"
                 [placeholder]="'text/plain'" />
+              <mat-hint class="flex flex-row items-center gap-1 mb-1">
+                Describes the content type of the data as a MIME type, see
+                <a
+                  class="link"
+                  externalLink
+                  href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types"
+                  >Common Types</a
```

Comment:
Because this is part as a sentence, maybe we should spell it lowercase?

Reply:
I think it looks great as it is:
![image](https://github.com/user-attachments/assets/d86e17eb-e15a-4919-b730-ec222c1335bb)